### ISSUE TITLE
Are 'mobile: swipe' and 'mobile: deepLink' correctly prefixed or should they be 'macos: swipe' and  'macos: deepLink'?

### ISSUE THREAD

**Issue by TestCentricity** [appium/appium-mac2-driver#302]
Seeking clarification of potential typos or cut-and-paste errors in `README.md` file.

Are `mobile: swipe` and `mobile: deepLink` correctly prefixed or should they be `macos: swipe` and  `macos: deepLink`? All other platform specific extension are prefixed with `macos: <methodName>`, so prefixes with `mobile:` are inconsistent with naming conventions.

**Comment by mykola-mokhnach** [comment]
you are right. They should be prefixed with `macos`. It's a copypaste mistake

**Comment by mykola-mokhnach** [comment]
would you like to create a PR and fix them?
